notebooks/llm-question-answering/llm-question-answering.ipynb

```diff
@@ -9,14 +9,14 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.                    minimum=1,
</pre><p>0 used in hf for disabling top_k sampling. can we use it for excluding this parameter from configuration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2062/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
done